packages/framework/src/Form/Transformers/NumericToMoneyTransformer.php

```diff
@@ -67,7 +65,5 @@ public function reverseTransform($value): ?Money
         if (is_float($value)) {
             return Money::createFromFloat($value, $this->floatScale);
         }
-
-        throw new TransformationFailedException('A string, a number or null must be provided.');
```

Comment:
Why did you removed these exceptions here?

Reply:
Because an exception call should not occur if used correctly. We have annotation for `$value` (* @param string | float | int | null $ value) and for all object we have condition.